### PR TITLE
Port WHO to TGUI

### PR DESCRIPTION
**Pull request by blackcrystall**
# About the pull request

Just QoL

# Explain why it's good for the game

Argghhh. just TGUI new panel? No benefits on round, but for visual and maybe a little bit server, because it don't generate every time for you who list

# Changelog

:cl: Blackcrystalic
qol: port who/staffwho to TGUI
/:cl:

### CODE REVIEW

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

File: code/controllers/subsystem/who.dm [hunk 1]
```diff
@@ -0,0 +1,306 @@
...
+		"text" = "Players playing as Yautja",
+	))
+
+	new_list_data["additional_info"] += list(list(
+		"content" = "Infected Predators: [additional_data["infected_preds"]]",
+		"color" = "#7ABA19",
+		"text" = "Players playing as Infected Yautja",
+	))
+
+	for(var/i in 10 to length(counted_factions) - 2)
```
Comment (harryob) [inline code/controllers/subsystem/who.dm:160]: why magic number here?

**Review by harryob — CHANGES_REQUESTED** [review]
just this, rest seems alright

**Review by Zonespace27 — DISMISSED** [review]
[0001-tgui-who-update.patch](https://github.com/user-attachments/files/16249978/0001-tgui-who-update.patch)

Apply this patch and it should be good to go

**Review by harryob — APPROVED** [review]

**Pull request merged**